Detectors/MUON/MCH/CTF/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Copyright CERN and copyright holders of ALICE O2. This software is distributed
+# under the terms of the GNU General Public License v3 (GPL Version 3), copied
+# verbatim in the file "COPYING".
+#
+# See http://alice-o2.web.cern.ch/license for full licensing information.
+#
+# In applying this license CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization or
+# submit itself to any jurisdiction.
+
+o2_add_library(MCHCTF
+               SOURCES src/CTFCoder.cxx src/CTFHelper.cxx
+               PUBLIC_LINK_LIBRARIES O2::DataFormatsMCH
+                                     O2::MCHBase
+                                     O2::DetectorsBase
+                                     O2::CommonDataFormat
+				                     O2::DetectorsCommonDataFormats
```

Comment:
could you untabify? This might be the problem in my old commit.

Reply:
yes, I will change that.